diff --git a/src/library/profile/Rprofile.unix b/src/library/profile/Rprofile.unix
index f008e68..7783d77 100644
--- a/src/library/profile/Rprofile.unix
+++ b/src/library/profile/Rprofile.unix
@@ -17,4 +17,4 @@ version <- Version()
 options(na.action="na.omit")
 options(printcmd=getenv("R_PRINTCMD"))
 options(papersize=getenv("R_PAPERSIZE"))
-if(interactive()) x11()
+
diff --git a/src/unix/system.c b/src/unix/system.c
index d25825f..45c509e 100644
--- a/src/unix/system.c
+++ b/src/unix/system.c
@@ -256,7 +256,7 @@ int main(int ac, char **av)
 #endif
 
 #ifdef linux
-	__setfpucw(_FPU_IEEE);
+	/* __setfpucw(_FPU_IEEE); */
 #endif
 
 #ifdef HAVE_LIBREADLINE
@@ -323,7 +323,7 @@ qask:
 #endif
 
 #ifdef linux
-	__setfpucw(_FPU_DEFAULT);
+	/* __setfpucw(_FPU_DEFAULT); */
 #endif
 
 	exit(0);
